Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -3951,6 +3980,84 @@ bool CStaticFunctionDefinitions::SetObjectMass ( CClientEntity& Entity, float fM
     return false;
 }
 
+bool CStaticFunctionDefinitions::SetObjectTurnMass ( CClientEntity& Entity, float fTurnMass )
+{
+    if ( fTurnMass >= 0.0f )
+    {
+        RUN_CHILDREN ( SetObjectTurnMass ( **iter, fTurnMass ) )
+
+        if ( IS_OBJECT ( &Entity ) )
+        {
+            CDeathmatchObject& Object = static_cast < CDeathmatchObject& > ( Entity );
+            Object.SetTurnMass ( fTurnMass );
+            return true;
+        }
+    }
+    return false;
+}
+
+bool CStaticFunctionDefinitions::SetObjectAirResistance ( CClientEntity& Entity, float fAirResistance )
+{
+    //if ( fAirResistance > 0.0f && fAirResistance <= 1.0f )

```

Comment:
Do you plan to uncomment that out before pushing ? (same for elasticity ?)

Reply:
I dont know yet how better